readthedocs/core/resolver.py

```diff
@@ -138,8 +140,8 @@ def resolve_domain(self, project, private=None):
 
         return getattr(settings, 'PRODUCTION_DOMAIN')
 
-    def resolve(self, project, require_https=False, filename='', private=None,
-                **kwargs):
+    def resolve(self, project, require_https=False, filename='',
+                query_params='', private=None, **kwargs):
```

Comment:
`query_param` should be `None` as default value instead of the empty string.

Reply:
I think `urlparser` only accepts an empty string, but I'll check